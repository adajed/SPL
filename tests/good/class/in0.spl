class Square
{
    int length;

    constr (int l)
    {
        length = l;
    }

    int getLength()
    {
        return length;
    }

    void setLength(int l)
    {
        length = l;
        return;
    }

    int area()
    {
        return length * length;
    }
}

int main()
{
    Square sq = new Square(1);
    sq.setLength(5);
    printInt(sq.getLength());
    printInt(sq.area());
    sq.setLength(7);
    printInt(sq.getLength());
    printInt(sq.area());

    return 0;
}

