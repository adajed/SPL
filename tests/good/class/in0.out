5
25
7
49
